src/app/cluster/cluster-details/alertmanager-config/component.ts

```diff
@@ -49,6 +59,18 @@ export class AlertmanagerConfigComponent implements OnDestroy {
     return _.isEmpty(this.alertmanagerConfig) && !this.isClusterRunning;
   }
 
+  displayAlertmanagerUILink(): boolean {
```

Comment:
Method name doesn't describe well what it actually does. Perhaps add 'should' at the beginning?

Reply:
sure. Done.